nvflare/private/fed/client/client_runner.py

```diff
@@ -305,10 +336,20 @@ def _process_task(self, task: TaskAssignment, fl_ctx: FLContext) -> Shareable:
 
         if filter_list:
             for f in filter_list:
+                filter_name = f.__class__.__name__
                 try:
                     reply = f.process(reply, fl_ctx)
+                except UnsafeJobError:
+                    self.log_exception(fl_ctx, f"UnsafeJobError from Task Result Filter {filter_name}")
```

Comment:
I think it might be good to have the reason print out:

```
except UnsafeJobError as e:
    self.log_exception(fl_ctx, f"UnsafeJobError ({secure_format_exception(e)}) from Task Data Filter {filter_name}")
```

Reply:
Done